crates/context/src/lib.rs

```diff
@@ -1009,4 +1011,73 @@ impl ContextManager {
     pub fn is_application_blob_installed(&self, blob_id: BlobId) -> EyreResult<bool> {
         self.blob_manager.has(blob_id)
     }
+
+    pub async fn propose(
+        &self,
+        context_id: ContextId,
+        signing_key: PublicKey,
+        proposal_id: ProposalId,
+        actions: Vec<u8>,
+    ) -> EyreResult<()> {
+        let handle = self.store.handle();
+
+        let actions: Vec<ProposalAction> = vec![]; // - How to deserialize actions?
+
+        let Some(context_config) = handle.get(&ContextConfigKey::new(context_id))? else {
+            //handle
+            return Ok(());
+        };
+
+        //How to get signer id?
+        let signer_id = SignerId::from_bytes(|buffer: &mut [u8; 32]| {
+            *buffer = [0; 32];
+            Ok(32)
+        })?;
```

Comment:
```suggestion
        let signer_id = signer_id.rt()?;
```

you can read `signing_key` from the `ContextIdentity` record in store

for example, see how this is done when inviting members to a context